docs/src/components/about/AboutHero.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { styled, keyframes } from '@mui/material/styles';
+import Section from 'docs/src/layouts/Section';
+import GradientText from 'docs/src/components/typography/GradientText';
+import TeamStatistics from 'docs/src/components/about/TeamStatistics';
+
+const teamPhotos = [
```

Comment:
Could we resize these images to match

```
    width: 400px;
    height: 300px;
```

?

If we want to optimize for desktop x2 (800 x 600), if we want to optimize for phones x3 (1200 x 900). Phone is likely the way to go.

There are images there is way above this. I imagine it's an important contributor to the issue of performance in https://pagespeed.web.dev/analysis/https-deploy-preview-38733--material-ui-netlify-app-about/0n4cjfs2zl

<img width="762" alt="Screenshot 2023-09-08 at 01 35 45" src="https://github.com/mui/material-ui/assets/3165635/7138942b-2142-4cbe-87fe-a7fe7389d20f">

vs. https://vercel.com/about

<img width="372" alt="Screenshot 2023-09-08 at 01 37 49" src="https://github.com/mui/material-ui/assets/3165635/eb1eaede-a435-4e03-bd47-2f996547fb7b">

(this surfaces a larger problem with our stack and UI tech)